package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "vscode_deno",
+  "name": "deno",
   "displayName": "%deno.displayName%",
   "description": "%deno.description%",
   "version": "1.24.0",
-  "publisher": "the deno authors",
+  "publisher": "denoland",
```

Comment:
revert

Reply:
https://github.com/denoland/vscode_deno/pull/35#discussion_r426184853

Reply:
the extension has not been published to the vscode market place with an official account, so we will keep this unchanged. When pr is merged into master, I will manually publish to justjavac.vscode-deno. 

When the extension is stable, I will mark justjavac.vscode-deno as deprecated and guide the user to install the official extension

Reply:
Ok, thanks for the explanation. I will revert my changes